src/webgpu/api/validation/encoding/encoder_open_state.spec.ts

```diff
@@ -103,6 +103,27 @@ const kRenderPassEncoderCommandInfo: {
 };
 const kRenderPassEncoderCommands = keysOf(kRenderPassEncoderCommandInfo);
 
```

Comment:
Oh and please remove the stale "TODO: Equivalent tests for GPURenderBundleEncoder." at the top of the file

Reply:
Ah, thank you for catching it! Removed.